src/main/java/spoon/support/compiler/jdt/JDTConstants.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (C) 2006-2019 INRIA and contributors
+ *
+ * Spoon is available either under the terms of the MIT License (see LICENSE-MIT.txt) of the Cecill-C License (see LICENSE-CECILL-C.txt). You as the user are entitled to choose the terms under which to adopt Spoon.
+ */
+package spoon.support.compiler.jdt;
+
+/**
+ * Extracts a subset of constants as defined in {@link org.eclipse.jdt.internal.compiler.lookup.TypeConstants}, since
+ * this class is only meant for internal use.
+ */
+public class JDTConstants {
+
+    public static final char[] MODULE_INFO_FILE_NAME = "module-info.java".toCharArray();
```

Comment:
Spoon uses tabs for tabulations. Right now the build fails because of the checkstyle:
> [ERROR] /home/travis/build/INRIA/spoon/src/main/java/spoon/support/compiler/jdt/JDTConstants.java:15: Indent must use tab characters [RegexpSinglelineJava]

Moreover you should provide a default private constructor for it:
> [ERROR] /home/travis/build/INRIA/spoon/src/main/java/spoon/support/compiler/jdt/JDTConstants.java:12:1: Utility classes should not have a public or default constructor. [HideUtilityClassConstructor]